orb/commands/@drupal.yml

```diff
@@ -134,7 +135,7 @@ drupal-helm-deploy:
             $reference_data_override \
             --namespace="$NAMESPACE" \
             --values '<<parameters.silta_config>>' \
-            --timeout 10m) 2>&1) || EXIT_CODE=$?
+            --timeout 15m) 2>&1) || EXIT_CODE=$?
```

Comment:
CircleCI has a default value for `no_output_timeout` of 10 minutes (see  https://circleci.com/docs/2.0/configuration-reference/#run). We might need to set it, otherwise CircleCI times out before helm does.

Reply:
based on deployment logs, there is not a 10 minute silence so it should be fine for now.

Reply:
I set the `no_output_timeout` now too.